packages/runtime/src/builder/serialization/types.ts

```diff
@@ -0,0 +1,517 @@
+import {
```

Comment:
`control-serialization.ts` was getting pretty long, so I moved the types into it's own file. This is the other big change.